src/vos/tests/vts_io.c

```diff
@@ -617,7 +617,7 @@ io_test_obj_fetch(struct io_test_args *arg, daos_epoch_t epoch,
 		return rc;
 	}
 
-	rc = vos_fetch_begin(arg->ctx.tc_co_hdl, arg->oid, epoch, 0, dkey,
+	rc = vos_fetch_begin(arg->ctx.tc_co_hdl, arg->oid, epoch, flags, dkey,
```

Comment:
'flags' needs be passed to 'non zero copy' case as well.

Reply:
good catch.   I should have done a sanity check on that...I added it last minute.